src/wrap-store/wrapStore.js

```diff
@@ -47,8 +47,18 @@ export default (store, {
       const action = Object.assign({}, request.payload, {
         _sender: sender
       });
-      dispatchResponder(store.dispatch(action), sendResponse);
-      return true;
+
+      let dispatchResult = null;
+
+      try {
+        dispatchResult = store.dispatch(action);
+      } catch (e) {
+        dispatchResult = Promise.reject(e.message);
+        console.error(e);
+      } finally {
```

Comment:
Half the battle of acquiring knowledge is proving things to yourself. :-)

From [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch):

```
The finally clause contains statements to execute after the try block and catch clause(s) execute, but before the statements following the try..catch..finally block. 
```

So basically, you can get rid of the `finally` block and just make it code that executes after the `try` or `catch` blocks execute and it's the same thing. :-)